README.md

```diff
@@ -49,6 +49,13 @@ docker-compose up
 Valkyrie stubs may also be used as a testing library, for example by starting a genericpam server programmatically as
 in this Valkyrie test suite [example](https://github.com/valkyrie-fnd/valkyrie/blob/main/provider/internal/test/suite.go#L51).
 
+### Test configuration
+The stubs contain an in-memory database. To configure the database, edit the datastore [config file](./datastore.config.yaml). Make sure that the appropriate player, provider, provider API key and game are present in the config. Otherwise add the values.
+
+To configure the valkyre module for the test, make sure that the provider is present [here](./valkyrie_config.yml) too.
+
+*Note*: The config files mentioned in this section are included in the image in the docker build, so there is no point changing them in runtime. If stubs is built and run as a go binary, the config files should be present in the execution directory and can be updated between each stop and start.
```

Comment:
no point.. **in** changing them at runtime?

Reply:
Fixed